doc/api/fs.markdown

```diff
@@ -989,6 +989,32 @@ The synchronous version of [`fs.writeFile()`][]. Returns `undefined`.
 
 Synchronous versions of [`fs.write()`][]. Returns the number of bytes written.
 
+## fs.mkdtemp(template, callback)
+
+Creates a unique temporary directory.
+
+The `fs.mkdtemp()` function generates a uniquely named temporary
+directory from template. The last six characters of template must be
+XXXXXX and these are replaced with a string that makes the directory
```

Comment:
@BridgeAR I don't think Python internally uses [mkdtemp(3)](http://man7.org/linux/man-pages/man3/mkdtemp.3.html), but it has its [own implementation](https://hg.python.org/cpython/file/3.5/Lib/tempfile.py#l345).